core/src/translate.rs

```diff
@@ -30,11 +39,31 @@ use {
         ReferentialAction as SqlReferentialAction, Statement as SqlStatement,
         TableConstraint as SqlTableConstraint, TableFactor, TableWithJoins,
     },
+    std::num::NonZeroUsize,
 };
 
 pub fn translate(sql_statement: &SqlStatement) -> Result<Statement> {
+    translate_with_params(sql_statement, std::iter::empty::<ParamLiteral>())
+}
+
+pub fn translate_with_params<I, P>(sql_statement: &SqlStatement, params: I) -> Result<Statement>
```

Comment:
It would be great to add a short doc comment here with a `# Errors` section explaining when this `Result` can fail.